vm/address/src/network.rs

```diff
@@ -2,9 +2,10 @@
 // SPDX-License-Identifier: Apache-2.0, MIT
 
 use super::{MAINNET_PREFIX, TESTNET_PREFIX};
+use serde::{Deserialize, Serialize};
 
 /// Network defines the preconfigured networks to use with address encoding
-#[derive(PartialEq, Eq, Copy, Clone, Debug, Hash)]
+#[derive(PartialEq, Eq, Copy, Clone, Serialize, Deserialize, Debug, Hash)]
```

Comment:
You don't need Serialize and Deserialize impls

Reply:
Yeah this was an oversight, was playing with address serialization